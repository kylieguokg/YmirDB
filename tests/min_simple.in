min no
set a
get a
min a
push a 0
get a
min a
set entry -1000 100 -1000 20 28 1000 -999
get entry
min entry
bye
