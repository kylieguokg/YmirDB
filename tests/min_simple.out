> no such key

> ok

> []

> empty entry

> ok

> [0]

> 0

> ok

> [-1000 100 -1000 20 28 1000 -999]

> -1000

> bye
